### ISSUE TITLE
PCB idea

### ISSUE THREAD

**Issue by oscarpeters** [robin7331/pixel-pump-mainboard#2]
Hi, I have looked at the design and found some improvements if I may :)

Does your manufacturer introduce extra costs for your Via construction? because the `annular ring` of some of your via's are less than `0.25mm` from the hole size.
![image](https://user-images.githubusercontent.com/42655076/216013295-06e4ac1d-ec38-4dc2-9b81-9fa8cdf74597.png)

other improvements:
![image](https://user-images.githubusercontent.com/42655076/216013407-6666776f-7911-4ec2-b634-962c610ac99b.png)
![image](https://user-images.githubusercontent.com/42655076/216013771-578b3b4c-7ece-4165-bf81-ec4faa982be9.png)
![image](https://user-images.githubusercontent.com/42655076/216014100-bc5522ce-94a7-4e12-8376-56af89ea180f.png)

Could make this a 3v3 plane:
![image](https://user-images.githubusercontent.com/42655076/216014495-0211a470-2884-4968-bbef-b7bef1410564.png)

lastly when making large quantities of these boards and maximum throughput from the pcb fab, I should no go further than 90 degress with copper traces, in these corners ething fluid can ingress and over time ruin the track.
![image](https://user-images.githubusercontent.com/42655076/216015433-85ea6268-45bf-45a8-a1ee-e9c9c75ff129.png)

**Comment by C44Supra** [comment]
JLC used to have a minimum annular ring of 0.3mm on the small run proto orders but has moved down to .13mm for 1oz and .2mm for 2oz copper. Those numbers vary however from manufacturer to manufacturer, and larger batch runs typically grant you as the designer some extra freedom. 

As for the acid trap thing, that hasn't been an issue for a few decades anymore, especially in professionally manufactured PCBs. If you're diying with etchant, try adding a bubbler and possibly rotating the pcbs during etching.

**Comment by oscarpeters** [comment]
Most other manufactures add extra costs of having an annular ring smaller than 0,25. The yield will be less the smaller the annular ring will be.

**Comment by C44Supra** [comment]
Not really. It's just a more involved process on 4 (or more) layer boards and little by little those processes are made available to lower count production runs too. Heck, can even get 4 layer boards at JLC for the price of a cheeseburger these days.

**Comment by oscarpeters** [comment]
Have you used other manufactures then jlc?

**Comment by robin7331** [comment]
Hey @oscarpeters and  @C44Supra, 

So far I had no problems (or a higher fee) due to anything related to the PCB design.
Also the development of this very first release is done. Meaning I am not planning on doing any additional EMI testing for this revision. That means I'll not change anything on the design if not super cricital. 

I highly appreciate your input and I'll leave this issue open until I am working on a new revision of this board.
